app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,50 @@
+<section class="container flex food-placeholder">
+<div class="food flex">
+<h1 class="title">Public Recipes</h1>
+
+<ul class="flex public">
+
+  <% @public_recipes.each do |recipe| %>
+    <li class="line flex">
+      <div id="<%= dom_id recipe %>">
+ 
+        <div>
+              <h2>
+                <%= recipe.name %> <br>
+                <span>
+                  By: 
+                  <%= recipe.user.name %>
+                </span>
+              </h2>
+        </div>
+
+      </div>
+
+        <div class="recipe-info flex">
+                  
+        </div>
+
+
+        <div class="recipe-cost flex">
+          <div>
+            <span>Total food items: <%= recipe.recipe_foods.count %> </span>
+          </div>
+          <div>
+            <span>Total price:            
+            <%= recipe.total_price %></span>
+          </div>
+      </div>
+      <br>
+    
+    </li>
+    <div class = "btn-right">
+      <p>
+        <%= link_to "Open Recipe", user_recipe_path(recipe.user, recipe), class: '' %>
+      </p>
+    </div>
+  <% end %>
+
+</ul>
+
+</div>
+</section>
```

Comment:
- [ ] I should see some public recipes on this page. It is [mentioned](https://github.com/belmeetmule/Recipe-App/pull/27#:~:text=Upon%20creating%20a%20recipe%20and%20making%20it%20public%2C%20it%20should%20show%20on%20the%20public%20recipe%20list.%20This%20is%20not%20the%20case%2C%20kindly%20fix%20this%20issue.) by the first reviewer also.
- [ ] For fix this when creating recipe you can add public recipe checkbox and if user click on it, recipe appear on the page. kindly fix it.
- [ ] OPTIONAL- It will better if public checkbox working on this page. Rİght now it is not working, only checkbox.
![image](https://user-images.githubusercontent.com/38635158/220131303-a996d9c4-f616-467a-be1d-20f7b954c397.png)